*** cugraph/python/setup.py-dist	2021-08-14 20:35:13.685504197 +0900
--- cugraph/python/setup.py	2021-08-14 20:36:32.028172177 +0900
***************
*** 131,137 ****
                    cuda_lib_dir,
                    os.path.join(os.sys.prefix, "lib")
                ],
!               libraries=['cudart', 'cusparse', 'cusolver', 'cugraph', 'nccl'],
                language='c++',
                extra_compile_args=['-std=c++17'])
  ]
--- 131,137 ----
                    cuda_lib_dir,
                    os.path.join(os.sys.prefix, "lib")
                ],
!               libraries=['cudart', 'cusparse', 'cusolver', 'cugraph', 'nccl', 'spdlog', 'fmt'],
                language='c++',
                extra_compile_args=['-std=c++17'])
  ]
***************
*** 150,156 ****
            # "Operating System :: OS Independent",
            "Programming Language :: Python",
            "Programming Language :: Python :: 3.6",
!           "Programming Language :: Python :: 3.7"
        ],
        # Include the separately-compiled shared library
        author="NVIDIA Corporation",
--- 150,158 ----
            # "Operating System :: OS Independent",
            "Programming Language :: Python",
            "Programming Language :: Python :: 3.6",
!           "Programming Language :: Python :: 3.7",
!           "Programming Language :: Python :: 3.8",
!           "Programming Language :: Python :: 3.9"
        ],
        # Include the separately-compiled shared library
        author="NVIDIA Corporation",
