*** cupy/install/build.py-dist	2021-08-27 00:06:42.067587693 +0900
--- cupy/install/build.py	2021-08-27 00:10:27.856921935 +0900
***************
*** 770,779 ****
          out = build_and_run(compiler, '''
          #include <stdio.h>
          #include <cugraph/version_config.hpp>
          int main(int argc, char* argv[]) {
!           printf("%d", CUGRAPH_VERSION_MAJOR * 10000
!                      + CUGRAPH_VERSION_MINOR * 100
!                      + CUGRAPH_VERSION_PATCH);
            return 0;
          }
          ''', include_dirs=settings['include_dirs'])
--- 770,779 ----
          out = build_and_run(compiler, '''
          #include <stdio.h>
          #include <cugraph/version_config.hpp>
+         #define CUGRAPH_VERSION_CODE \
+                             210803
          int main(int argc, char* argv[]) {
!           printf("%d", CUGRAPH_VERSION_CODE);
            return 0;
          }
          ''', include_dirs=settings['include_dirs'])
